grepos/app/src/main/java/net/boondockradio/grepos/adapter/RepositoryAdapter.java

```diff
@@ -0,0 +1,125 @@
+package net.boondockradio.grepos.adapter;
+
+import net.boondockradio.grepos.R;
+import net.boondockradio.grepos.dto.Repository;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import java.util.List;
+
+public class RepositoryAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+    private enum  ViewType {
+        Progress(0),
```

Comment:
instead of declaring enum index value as `type`, enum has its own index value called `ordinal`, so maybe you could use  that? 👀  what do you think? http://qiita.com/amay077/items/097f54b7dee586fadc99